### PR TITLE
[Filev] Initial Revision of the Cog

### PR DESCRIPTION
**Pull request by Kreusada**
### Type

- [ ] Bugfix
- [ ] Enhancement
- [x] New feature
- [ ] Documentation

### Description of the changes

The Filev cog

### CODE REVIEW

File: yamlscanner/scanner.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+            return "{} {}".format("\N{BALLOT BOX WITH CHECK}\N{VARIATION SELECTOR-16}", text)
+        
+        def cross(text) -> str:
+            return "{} {}".format("\N{CROSS MARK}", text)
+
+        if ctx.message.attachments[0]:
+            # attachments will take priority
+            file = ctx.message.attachments[0]
+            filename = file.filename
+            if not filename.split('.')[-1] in ("yaml", "yml", "mir"):
```
Comment (Just-Jojo) [inline yamlscanner/scanner.py:69]: As this is the only place that `filename` appears it doesn't make too much sense to make a separate variable for it (Optional)

**Review by Just-Jojo — APPROVED** [review]
LGTM. One that that *could* be changed but doesn't need to

**Comment by Kreusada** [comment]
Thanks, I often do that lol

**Comment by Kreusada** [comment]
I no longer want to work on this.

**Pull request closed without merging**